### ISSUE TITLE
Test Performance 

### ISSUE THREAD

**Issue by vinodkkurmi** [as641651/Faster-RCNN-Torch#1]
hi  Is there any problem in evaluation code?? I am getting around 94 mAP and 96 recall value. Are you doing is with train data ???

**Comment by as641651** [comment]
No. I evaluate with PASCAL VOC07 test  set

**Comment by vinodkkurmi** [comment]
@as641651  Hi Thanks for your response.
Can you tell at what iteration you got the mAP 68.2% ????

**Comment by ataldim** [comment]
Hello.
I got 68.2% mAP when testing with the pre-trained faster-rcnn VGG16 model, which as I understand from the description was not trained with this implementation. 
When training with vgg16 for initialiazation, I get a mAP of only 57-58% with the provided config (70000 iterations). Further training didn't  increase the result. Do you get a similar mAP value?

**Comment by as641651** [comment]
@ataldim Yes I get something similar when initialized with VGG16. In the original  py-faster-rcnn, they append the dataset with flipped images, which I have not done here.  I ll mostly take this project up in December again and add this functionality.

**Comment by as641651** [comment]
@vinodkkurmi  The test performance is for the mode trained with the python version of faster rcnn in the original paper. Further training actually did not improve yet, probably because the training set is just half of the original version without flipped images

**Comment by vinodkkurmi** [comment]
@as641651  Thanks for your response. It means if we train using pre trained VGG16 model (as py-faster rcnn ) using this code, we get only 57-57%map . If we just evaluate py-faster rcnn model it gives the 68.2%map ???
Am I right?
If this is true, then your code is far away from the actual result.It means there is some problem in coding part. Can you share what part is missing so we can include that and try to achieve the actual performance?

**Comment by as641651** [comment]
@vinodkkurmi In the original py-faster-rcnn, they append the dataset with flipped images, which I have not done here. Doing this will double the training set.

**Comment by gfsliumin** [comment]
Hi, in ApplyBoxTransform.lua, given a bounding box with center (xa, ya), width wa, and height ha,
and given offsets (tx, ty, tw, th), compute the new bounding box (x, y, w, h) as:
x = tx * wa + xa
but you give the gradients (dx, dy, dw, dh) on the output the gradients on the inputs as:
dtx = wa * dx
Is this right? 
why not dtx=dx/wa?
